engines/docker/generated_configschema.go

```diff
@@ -0,0 +1,47 @@
+// This source code file is AUTO-GENERATED by github.com/taskcluster/jsonschema2go
+
+package docker
+
+import "github.com/taskcluster/taskcluster-worker/runtime"
+
+type (
+	// Config applicable to docker engine
+	Config struct {
+
+		// Root Volume blah blah
+		RootVolume string `json:"rootVolume"`
+	}
+)
+
+func ConfigSchema() runtime.CompositeSchema {
+	schema, err := runtime.NewCompositeSchema(
+		"config",
+		`
+		{
+		  "$schema": "http://json-schema.org/draft-04/schema#",
+		  "additionalProperties": false,
+		  "description": "Config applicable to docker engine",
+		  "properties": {
+		    "rootVolume": {
+		      "description": "Root Volume blah blah",
+		      "title": "Root Volume",
+		      "type": "string"
+		    }
+		  },
+		  "required": [
+		    "rootVolume"
+		  ],
+		  "title": "Config",
+		  "type": "object"
```

Comment:
Thinking about this some more, we don't really have a place to put general purpose go packages. Should we keep taskcluster-base-go for general purpose packages? For example I'm using jsontest in several projects (generic-worker, taskcluster-client-go, taskcluster-worker, ...). I could stick it in the client, but it is not client-specific.

We could rename taskcluster-base-go to just 'go' or 'go-libs' - then we'd have e.g.:

* `github.com/taskcluster/go/jsonutil`
* `github.com/taskcluster/go/text`
* `github.com/taskcluster/go/atomics`
* `github.com/taskcluster/go/atomics`

I think it would be overkill to place each package in its own repository, so a single repository for general utility packages seems like a reasonable compromise.